index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/normalize.css">
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="icon" href="./images/favicon.png" type="image/x-icon">
+    <title>Nova Page</title>
+</head>
+
+<body>
+    <header class="header container">
+        <div class="header__nav box">
+            <a class="header__logo" href="/"><img src="./images/logo.svg" alt="logo nova"></a>
+
+            <nav class="header__menu">
+                <ul class="header__menu-list">
+                    <li><a class="header__menu-link" href="">The Team</a></li>
+                    <li><a class="header__menu-link" href="">Pricing</a></li>
+                    <li><a class="header__menu-link" href="">Features</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="header__cta box">
```

Comment:
Wrzuciłbym znacznik `article`, bo to jedna spójna treść, która znajduje się wew.